### DISCUSSION TITLE
Codemagic IOS Runner build issue

### DISCUSSION THREAD

**Discussion by aminEspark** [codemagic-ci-cd/codemagic-docs#2897] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1 We are using the default workflow for building the iOS app (i.e., we are not using the codemagic.yaml configuration).
2 The iOS project includes two targets:
  Runner: The main iOS application.
  Notification Extension: Used to display images in push notifications.
3 When attempting to build the iOS app, we encounter an error stating that the Runner target is not found. I've also attached an image of the error from Codemagic.
![Code magic](https://github.com/user-attachments/assets/668089f5-c279-41c5-acb5-105c597ceeb4)

### Expected results

The expected result is a fully functional iOS app that supports displaying images in push notifications without any build errors.

### Actual results

During the iOS app build process, at the 5th step of installing dependencies, we encounter an error stating that the Runner target cannot be found in the repository.

### Build id (optional)

676410f27539d5477abb0140

**Comment by icarusdust** [comment]
Hi @aminEspark, I see that you have access to the paid feature chat widget in the Codemagic web app, please use it for a quicker support option. Currently, due to the holiday season, we expect delays in responses in the public support channels.

**Reply by aminEspark** [reply]
@icarusdust Thank you for your time.

**Reply by icarusdust** [reply]
As you have an ongoing thread with the support team in Intercom, I will go ahead and close it here.

**Reply by sadespresso** [reply]
@aminEspark Hello there! Were you able to get your issue resolved? I'm having the exact same issue, it'd be really nice if you could share how you managed to resolve :D

**Reply by dam-ease** [reply]
Hello @sadespresso,
One of the suggestions here was to upgrade to an earlier version of Flutter. However, based on your comment in another thread, it seems you mentioned that you are using the latest version of Flutter. Therefore, your issue may be different from this one. Did the solution here help you - https://github.com/orgs/codemagic-ci-cd/discussions/2907#discussioncomment-12339677? If not, could you please open a discussion at https://github.com/orgs/codemagic-ci-cd/discussions/new?category=q-a and include your build ID?

**Reply by sadespresso** [reply]
Hello @dam-ease, thank you for the reply!

Your reply on the other thread helped, adding 'flutter pub get' to the post-clone script made it work. This was on Flutter 3.29 (latest stable)

I should've replied there 😅

Thank you for your support 😊
